hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerDoubleBuffer.java

```diff
@@ -147,7 +160,11 @@ private void flushTransactions() {
             });
 
             long startTime = Time.monotonicNowNanos();
-            omMetadataManager.getStore().commitBatchOperation(batchOperation);
+            try (Scope s2 = TracingUtil.importAndCreateScope(
+                "DB-commitWriteBatch", lastTraceId.get())) {
```

Comment:
One more question, here for DB commit Batch we are using last trace id, so DB Batch commit span will be linked with the last operation in the queue?

Reply:
Yes. We are effectively piggy back commitWrite to the last Op that triggered the flush. The other choice is to do another loop of the ready buffers to add it to each of the individual Ops, which will have higher cost.

Reply:
Got it.